spark_pipeline_framework/transformers/send_automapper_to_fhir/v1/automapper_to_fhir_transformer.py

```diff
@@ -66,6 +66,8 @@ def __init__(
         sort_data: Optional[Dict[str, Dict[str, Any]]] = None,
         partition_by_column_name: Optional[str] = None,
         enable_repartitioning: bool = True,
+        debug: bool = False,
+        func_get_preprocessed_path: Optional[Callable[[str, str], str]] = None,
```

Comment:
There should be consistency in function arguments like we can use func_get_debug_file_path here.